UltraStar Play/Assets/Common/UI/UIUtils.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public static class UIUtils
+{
+    // Returns a nicely readable string from a filename, e.g. "theme_blue" will become "Theme Blue"
+    public static string BeautifyString(string input)
```

Comment:
- Beautiful is relative. Please name the method `SnakeCaseToTitleCase` or similar.